Lib/dataclasses.py

```diff
@@ -6,6 +6,10 @@
 import keyword
 import builtins
 
+from functools import wraps
+from threading import get_ident
+
```

Comment:
I'd rather keep the same style import as above: just import the entire module, and use (for example) `functools.wraps` below. Then just insert the modules in the list above and get rid of the blank lines.

Also: `reprlib` uses `_threading` to get to `get_indent`. Why not do the same here?